integration_tests/integration/barcode/selectEnvelopeSize.spec.ts

```diff
@@ -0,0 +1,40 @@
+import Page from '../../pages/page'
+import FindRecipientByPrisonNumberPage from '../../pages/barcode/findRecipientByPrisonNumber'
+import CreateNewContactPage from '../../pages/barcode/createNewContact'
+import ReviewRecipientsPage from '../../pages/barcode/reviewRecipients'
+import ChooseBarcodeOptionPage from '../../pages/barcode/chooseBarcodeOption'
+import SelectEnvelopeSizePage from '../../pages/barcode/selectEnvelopeSize'
+import PrintCoversheetsPage from '../../pages/barcode/printCoversheets'
+
+context('Select envelope size', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubAuthToken')
+    cy.task('stubRequestLink')
+    cy.task('stubVerifyLink')
+    cy.visit('/link/verify-link?secret=a-valid-secret')
+    cy.task('stubGetPrisonRegister')
+    cy.visit('/barcode/find-recipient')
+    Page.verifyOnPage(FindRecipientByPrisonNumberPage).submitWithValidPrisonNumber()
+    Page.verifyOnPage(CreateNewContactPage).submitWithValidValues()
+    Page.verifyOnPage(ReviewRecipientsPage).prepareBarcodes()
+    Page.verifyOnPage(ChooseBarcodeOptionPage).continueToCoversheet()
+  })
+
+  it('should redisplay form with errors given no envelope size selected', () => {
+    const selectEnvelopeSizePage = Page.verifyOnPage(SelectEnvelopeSizePage)
+
+    selectEnvelopeSizePage.submitWithoutSelectingEnvelopeSize()
+
+    selectEnvelopeSizePage.hasErrorContaining('Select an option')
+    selectEnvelopeSizePage.signOut().click()
```

Comment:
We should probably look at this a bit more at some stage, but similar to what you had to do yesterday, I also had to sign out after each test because otherwise it remembers the session data 🤷‍♂️ 

Weirdly (and I'm not sure how yours works, but it does) .... the `Page.signOut()` method only selects the button, it doesnt actually click it !!!
We use the `signOut` method in a number of places and apparently dont click it!!, so I dont know what it does / how it works .... but I do know I needed it here 🤷